sensing/autoware_pointcloud_preprocessor/include/autoware/pointcloud_preprocessor/concatenate_data/concatenate_pointclouds.hpp

```diff
@@ -139,9 +140,7 @@ class PointCloudConcatenationComponent : public rclcpp::Node
   // XmlRpc::XmlRpcValue input_topics_;
   std::vector<std::string> input_topics_;
 
-  /** \brief TF listener object. */
-  std::shared_ptr<tf2_ros::Buffer> tf2_buffer_;
-  std::shared_ptr<tf2_ros::TransformListener> tf2_listener_;
+  std::shared_ptr<autoware::universe_utils::StaticTransformBuffer> static_tf_buffer_{nullptr};
```

Comment:
unique?

Reply:
There seem to be more occurrences of this in case you accept the suggestion

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/8174/commits/c55991a5193aaaa465bb5fd45e087d01136ba8ca